lib/gh-checks.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const chalk = require('chalk');
+const execa = require('execa');
+const ghGot = require('gh-got');
+const hostedGitInfo = require('hosted-git-info');
+const pTimeout = require('p-timeout');
+const pWaitFor = require('p-wait-for');
+const terminalLink = require('terminal-link');
+const util = require('./util');
+
+const THIRTY_SECONDS = 1000 * 30;
+const THIRTY_MINUTES = 1000 * 60 * 30;
+
+module.exports = task => {
+	return execa.stdout('git', ['cherry'])
+		.then(stdout => {
+			if (stdout === '') {
+				return;
+			}
+
+			return execa('git', ['push']);
+		})
+		.then(() => execa.stdout('git', ['rev-parse', 'HEAD']))
+		.then(commit => {
```

Comment:
`commit` => `latestCommit` would be more descriptive.

Reply:
Changed in 8bff67727f75b16b34fc9afd5279a5eb82c3167b.